tests/fast-integration/package.json

```diff
@@ -30,7 +30,7 @@
     "mocha-multi-reporters": "^1.5.1"
   },
   "dependencies": {
-    "@kubernetes/client-node": "0.13.0",
+    "@kubernetes/client-node": "0.15.1",
```

Comment:
the old version of this library appears to have a bug with getting container logs, that is why I need an update here

```
/home/wozy/projects/go/src/github.com/kyma-project/kyma/tests/fast-integration/node_modules/@kubernetes/client-node/dist/log.js:31
                    done(null);
                    ^

TypeError: done is not a function
    at Request._callback (/home/wozy/projects/go/src/github.com/kyma-project/kyma/tests/fast-integration/node_modules/@kubernetes/client-node/dist/log.js:31:21)
    at Request.self.callback (/home/wozy/projects/go/src/github.com/kyma-project/kyma/tests/fast-integration/node_modules/request/request.js:185:22)
    at Request.emit (node:events:394:28)
    at Request.<anonymous> (/home/wozy/projects/go/src/github.com/kyma-project/kyma/tests/fast-integration/node_modules/request/request.js:1154:10)
    at Request.emit (node:events:406:35)
    at IncomingMessage.<anonymous> (/home/wozy/projects/go/src/github.com/kyma-project/kyma/tests/fast-integration/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (node:events:513:28)
    at IncomingMessage.emit (node:events:406:35)
    at endReadableNT (node:internal/streams/readable:1343:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```